qa/L0_perf_resnet/test.sh

```diff
@@ -89,29 +90,33 @@ $CAFFE2PLAN -h -b ${STATIC_BATCH} \
 
 # Tests with each "non-optimized" model
 for MODEL_NAME in $MODEL_NAMES; do
-    REPO=`pwd`/tensorrt_models && [ "$MODEL_NAME" != "$TRT_MODEL_NAME" ] && \
-        REPO=$REPODIR/perf_model_store
-    FRAMEWORK=$(echo ${MODEL_NAME} | cut -d '_' -f 3)
-    MODEL_NAME=${MODEL_NAME} \
-              MODEL_FRAMEWORK=${FRAMEWORK} \
-              MODEL_PATH="$REPO/${MODEL_NAME}" \
-              STATIC_BATCH_SIZES=${STATIC_BATCH} \
-              DYNAMIC_BATCH_SIZES=${DYNAMIC_BATCH} \
-              INSTANCE_COUNTS=${INSTANCE_CNT} \
-              bash -x run_test.sh
+    for PROTOCOL in $PROTOCOLS; do
+        REPO=`pwd`/tensorrt_models && [ "$MODEL_NAME" != "$TRT_MODEL_NAME" ] && \
+            REPO=$REPODIR/perf_model_store
+        FRAMEWORK=$(echo ${MODEL_NAME} | cut -d '_' -f 3)
+        MODEL_NAME=${MODEL_NAME} \
+                MODEL_FRAMEWORK=${FRAMEWORK} \
+                MODEL_PATH="$REPO/${MODEL_NAME}" \
+                STATIC_BATCH=${STATIC_BATCH} \
+                PERF_CLIENT_PROTOCOL=${PROTOCOL} \
```

Comment:
PERF_CLIENT_PROTOCOL -> PROTOCOL

Reply:
I kept the name as PERF_CLIENT_PROTOCOL to keep it homogeneous with the rest of the benchmark tests.